src/Simulation/qdk_sim_rs/benches/c_api_benchmark.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//! This set of benchmarks exercises the open systems simulator exclusively via
```

Comment:
This benchmarking is really cool! Not for this PR, but I wonder if there is a way we could generalize some of this (or at least put a guide somewhere in the repo as a readme) so that future Rust components can easily include their own benchmarks?

Reply:
Thanks! Maybe Rust benchmarking best practices would be something to add to the repo wiki? Would love to figure out the best way to pick up benchmarks from the CI artifacts as part of writing that guidance too (+@cesarzc for discussion here as well).